redwood-treehouse-host/src/commonTest/kotlin/app/cash/redwood/treehouse/FakeZiplineTreehouseUi.kt

```diff
@@ -64,6 +67,8 @@ class FakeZiplineTreehouseUi(
     val result = host.addOnBackPressedCallback(object : OnBackPressedCallbackService {
       override var isEnabled = isEnabled
 
+      override val enabledChangedEvents: Flow<Unit> = emptyFlow()
+
```

Comment:
I want to actually test the toggling of `isEnabled` and make sure everything propagates correctly. However, I don't want to copy [the production code logic](https://github.com/cashapp/redwood/blob/d6bf86806629727b5f7f0f80b045a8c5faf433dd/redwood-treehouse-guest/src/commonMain/kotlin/app/cash/redwood/treehouse/treehouseCompose.kt#L140-L163) to this fake, as I want to be testing the production code and not a fake.